montecarlo/run_sim.sh

```diff
@@ -359,7 +359,7 @@ rm ${FAX_FILENAME}.*  # Peak-by-peak file with all photoionization info
 
 
 # hax stage
-HAX_TREEMAKERS="Basics Fundamentals DoubleScatter LargestPeakProperties TotalProperties Extended"
+HAX_TREEMAKERS="Basics Fundamentals Corrections DoubleScatter LargestPeakProperties TotalProperties Extended"
```

Comment:
I think it must come before Basics according to https://github.com/XENON1T/hax/pull/109